index.html

```diff
@@ -6,11 +6,23 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Document</title>
     <link rel="stylesheet" href="style.css">
-    </head>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet">
+</head>
 <body>
-    <header>
-        <img src="" alt="">
+    <header id="toolbar">
+        <img src="images/ic_menu.svg" alt="Menu">
+        <a href=""></a> 
     </header>
-    <h1>Felipe Haybar</h1>
+        <h1>Felipe Haybar</h1>
+    <section id="headline">
+        <p>Hello! I am a software developer! I can help you build a product, feature or website. Taka a look of my works.
+        <br>If you like what you see and have a project you need coded, don’t hesitate and contact me.</p>
+    </section>
+    <button class="blackBoton">
+        <a class="textBoton" href="#">Start collaboration</a>
+    </button>
+    <img src="images/image_geometry_2.png" alt="geometric art" class="art">
```

Comment:
Kindly add this image as a background using CSS. If an image is not part of the content (aesthetic reasons) use CSS background-image. :eyes: :eyes:

Reply:
I´ll aply inmediatly